### DISCUSSION TITLE
RuntimeError: CUDA error: device-side assert triggered

### DISCUSSION THREAD

**Discussion by zhmingzhaung** [zai-org/ChatGLM3#1051] [Q&A] [answered]
请问大家，在推理过程中，加载模型后报错如下：
RuntimeError: CUDA error: device-side assert triggered
CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions.
这个应该怎么解决呀。
![1711612335772](https://github.com/THUDM/ChatGLM3/assets/152263368/6aae43e2-03eb-4071-bc8e-cc0dd0d9498a)

**Comment by ecsfu** [comment, marked as answer]
参考这个   https://github.com/THUDM/ChatGLM3/discussions/393#discussioncomment-8963996

**Reply by zhmingzhaung** [reply]
成功解决，谢谢！
